imports/plugins/core/layout/client/templates/layout/admin/admin.html

```diff
@@ -21,11 +21,11 @@
       {{> Template.dynamic template=template}}
```

Comment:
Do we even need this file anymore? This should no longer be used, as it's been replaced with React components `app.js` (located in core/ui package) and `coreLayout.js`.

Reply:
I think not, as i've stated in the getFooterComponent helper function. I'm going to delete this file.